modules/blender/hm.nix

```
@@ -1,46 +1,52 @@
 {
-  config,
+  mkTarget,
   lib,
-  pkgs,
   ...
 }:
-{
-  options.stylix.targets.blender.enable =
-    config.lib.stylix.mkEnableTarget "blender" true;
+mkTarget {
+  name = "blender";
+  humanName = "Blender";
+
+  extraOptions.themeBody = lib.mkOption {
+    type = lib.types.lines;
+    default = "";
+    internal = true;
+  };
 
-  config =
-    lib.mkIf (config.stylix.enable && config.stylix.targets.blender.enable)
+  configElements = [
+    (import ./color-theme.nix)
+    (import ./font-theme.nix)
```

Could you test whether https://github.com/nix-community/stylix/pull/1523 works with this PR as:

```suggestion
    ./color-theme.nix
    ./font-theme.nix
```

---

Works identically :+1:

---

> Works identically 👍

In that case, this PR should be merged after https://github.com/nix-community/stylix/pull/1523.

---

> > Works identically 👍
>
> In that case, this PR should be merged after #1523.

This has been merged in commit c700d41bb8ee ("stylix: allow mkTarget's configuration arguments to receive paths (#1523)"). Can you rebase on top of that commit and apply the previously proposed change?